packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -224,6 +226,8 @@ $expand-less-icon: map-merge(
 
     &[aria-expanded] {
       span {
+        display: inline-block;
+
         &::after {
           position: absolute;
           top: 50%;
```

Comment:
Im curious if it would make more sense if we moved these lines to `usa-nav__primary.usa-accordion__button.span` on line 300 instead of `[aria-expanded]. Seems like it might be a bit more descriptive but it is further away from the other `aria-expanded=true/false` styles which are relevant to this work.

https://github.com/uswds/uswds/blob/05e4ee234628ca6f45a3ca76ef1e470a02a0add3/packages/usa-nav/src/styles/_usa-nav.scss#L307-L315